Campus-iOS/LoginComponent/Views/TokenConfirmationView.swift

```diff
@@ -147,6 +251,9 @@ struct TokenConfirmationView: View {
         .task {
             await switchSteps()
         }
+        .sheet(isPresented: $showTUMOnline) {
+             SFSafariViewWrapper(url: URL(string: "https://www.campus.tum.de")!).edgesIgnoringSafeArea(.bottom)
```

Comment:
Can you directly direct to the token view in campus online?

Reply:
✅